cmd/crowdsec-cli/ban.go

```diff
@@ -84,7 +87,109 @@ func simpleBanToSignal(targetIP string, reason string, expirationStr string, act
 	return signalOcc, nil
 }
 
+func filterBans(bans []map[string]string) ([]map[string]string, error) {
+	// fmt.Printf("source,ip,reason,bans,action,country,as,events_count,expiration\n")
+	// 	for _, rm := range ret {
+	// 		fmt.Printf("%s,%s,%s,%s,%s,%s,%s,%s,%s\n", rm["source"], rm["iptext"], rm["reason"], rm["bancount"], rm["action"], rm["cn"], rm["as"], rm["events_count"], rm["until"])
+	// 	}
+	var retBans []map[string]string
+
+	for _, ban := range bans {
+		var banIP net.IP
+		var banRange *net.IPNet
+		var keep bool = true
+		var err error
+
+		//try to parse ip or range from ban
+		if ban["iptext"] != "" {
+			if strings.Contains(ban["iptext"], "/") {
+				log.Debugf("%s is a range", ban["iptext"])
+				banIP, banRange, err = net.ParseCIDR(ban["iptext"])
+				if err != nil {
+					log.Warningf("failed to parse '%s' as a range : %s", ban["iptext"], err)
```

Comment:
Do we want to print this to the user in case of this happening ?
If yes, maybe we should give more details about where this IP come from ?
```
failed to parse '%s' ip found in the backend database as a range ... 
```